client/src/app/components/forms/code-input-form/code-input-form.component.scss

```diff
@@ -25,7 +25,7 @@
 	font-size: 2em;
 	text-align: center;
 	padding-bottom: 5px;
-	font-family: auto;
+	font-family: Spectral;
```

Comment:
Good practice to add a backup font here even though we know its available. Something like:

`font-family: Spectral serif`

Reply:
would serif be the backup font here? or

Reply:
pushed up